apps/api/.env-template

```diff
@@ -362,3 +362,17 @@ METRICS_INSTANCE_ID=dev_local
 #METRICS_COLLECT_HTTP_REQUEST_DURATION=
 #METRICS_COLLECT_HTTP_CONTENT_LENGTH=
 #METRICS_COLLECT_JOBS_DURATION=
+
```

Comment:
I would remove this config, it unlikely it will ever change and we need to pull this API config withg whole other stuff into the CLI just for of these constansts